retrier/retrier.go

```diff
@@ -89,7 +90,15 @@ func (r *Retrier) RunFn(ctx context.Context, work func(ctx context.Context, retr
 				return ret
 			}
 
-			timer := time.NewTimer(r.calcSleep(retries))
+			var err *errWithBackoff
+			var backoff time.Duration
+			if errors.As(ret, &err) {
+				backoff = err.backoff
```

Comment:
If the response gives us a backoff value, does it still make sense to add some jitter to that? I'm not honestly sure...

Reply:
I agree with eapache's point.
I don't think Jitter should be unnecessarily considered by the client when given by Retry-After.
The client should add the Jitter because it is the same program or could be executed at the same time, and the Retry-After reverses the timing decision.